src/main.js

```diff
@@ -80,19 +106,19 @@ let state;
 
 // is there a timer here? I dunno!
 let mainloop = function (repairsList) {
-  personinfo.personinfo(
-    null, // tour
-    function (gottenState) {
-      actions.log("about to runAll");
-      console.log(gottenState);
-      state = gottenState;  // leak it
-      runner.runAll(repairsList, state,
-        function () { actions.log("runAll callback"); }
-    );}
+  runner.runAll(repairsList,
```

Comment:
should check that repairsList is valid.
- names unique   #38
- all are valid recipes.